com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDAdditionalCameraData.cs

```diff
@@ -8,6 +8,7 @@ namespace UnityEngine.Rendering.HighDefinition
     /// Holds the physical settings set on cameras.
     /// </summary>
     [Serializable]
+    [Obsolete("Properties have been migrated to Camera class")]
```

Comment:
```suggestion
    [Obsolete("Properties have been migrated to Camera class", false)]
```
The way to obsolete a public API is: 
APIs (i.e,  public -> obsolete(false) -> obsolete(true) -> removal)

Also, URP has a **.Deprecated.cs** where this stuff is added, basically you do the struct partial, and move everthing obsolete there, in this case if the full struct is obsolete we simply move the struct there. It helps doing the path to obsolete